.github/workflows/push.yml

```
@@ -452,15 +456,22 @@
+          ./scripts/ci/10_prepare-docker-deps.sh
+
   tests:
     needs:
       - prepare
-    if: needs.prepare.outputs.TEST_CHUNKS
+    if: ${{ needs.prepare.outputs.TEST_CHUNKS && needs.prepare.outputs.DEPLOY_FEATURE != 'true' }}
     runs-on: arc-runners
     timeout-minutes: 45
     env:
       AFFECTED_PROJECTS: ${{ matrix.projects }}
       NX_BASE: ${{ needs.prepare.outputs.NX_BASE }}
```

actionlint (1.7.4)

property "nx_base" is not defined in object type {build_chunks: string; deploy_feature: string; docker_matrix: string; docker_tag: string; images: string; last_good_build_docker_tag: string; node-modules-hash: string; node_image_version: string; test_chunks: string; unaffected: string}

(expression)